src/System.Private.Xml/src/System.Private.Xml.csproj

```diff
@@ -8,6 +8,7 @@
     <AllowUnsafeBlocks>true</AllowUnsafeBlocks>
     <NoWarn>$(NoWarn),649</NoWarn>
     <DefineConstants Condition="'$(TargetGroup)' == 'uap'">$(DefineConstants);uap</DefineConstants>
```

Comment:
It would be useful to define `FEATURE_XXX` defines for this instead, so that we do not have to redo all ifdefs every time we change our mind about the exact config used.